docs/documentation/maya/simshape.md

```diff
@@ -136,46 +159,69 @@ To remove any of these meshes from simshape follow this procedure:
 
 ## Attribute Editor Template
 
 <figure markdown>
   ![simshape editor first part](images/attribute_editor_part_one_simshape.png) 
-  <figcaption>Figure 1: Simshape Attribute Editor</figcaption>
+  <figcaption><b>Figure 2:</b> AdnSimshape Attribute Editor</figcaption>
 </figure>
 
 <figure markdown>
   ![simshape editor second part](images/attribute_editor_part_two_simshape.png)
-  <figcaption>Figure 2: Simshape Attribute Editor (Advanced Settings)</figcaption>
+  <figcaption><b>Figure 3:</b> AdnSimshape Attribute Editor (Advanced Settings)</figcaption>
 </figure>
 
+<figure markdown>
+![skin editor debug menu](images/attribute_editor_simshape_debug.png)
+<figcaption><b>Figure 4:</b> AdnSimshape Attribute Editor (Debug menu)</figcaption>
+</figure>
+
+## Debugger
+
+To better visualize deformer constraints and attributes in the Maya viewport there is the option to enable the debugger, found in the dropdown menu labeled *Debug* in the Attribute Editor.
+
+To enable the debugger the *Debug* checkbox must be marked. To select the specific feature you would like to visualize, choose it from the list provided in *Features*.
+
+### Debug features
+
+The features that can be visualized with the debugger in the Skin deformer are:
+
+ - **Collision Constraints**: For each vertex, a line will be drawn from the mesh to the closest point of a collider.
+    - The debug lines will only be displayed in case collisions are enabled and colliders have been set up.
+ - **Muscle Fibers**: For each vertex, a line will be drawn showing the direction of the muscle fibers.
+    - The debug lines will only be displayed in case muscle activations have been enabled with an Adonis Muscle Patches file.
+
+Enabling the debugger and selecting one of these constraints will draw lines from the influenced vertices in the simulated mesh to their corresponding reference vertices. 
+
+<!-- We might want to add a screenshot here of a debug functionality like fibers -->
+
 # Advanced
 
 ## Muscle Activations
-Simshape can emulate the behaviour of facial muscles by computing the muscle activation directly on the vertices of the skin geometry. The activation of the vertices is an advanced and optional feature that can work in two modes: from **muscle patches data** or from **plug values**.
+AdnSimshape can emulate the behaviour of facial muscles by computing the muscle activation directly on the vertices of the skin geometry. The activation of the vertices is an advanced and optional feature that can work in two modes: from **muscle patches data** or from **plug values**.
 
 <figure markdown>
   ![Activation modes from attribute editor](images/activation_modes.png)
-  <figcaption>Figure 1: Activation Modes switch exposed in the Attribute Editor</figcaption>
+  <figcaption><b>Figure 5:</b> Activation Modes switch exposed in the Attribute Editor</figcaption>
 </figure>
 
 !!! abstract "Activations Modes"
```

Comment:
Replace `[this section](#generate-muscle-patches)` with `[generate muscle patches section](#generate-muscle-patches)`

Reply:
Done